### ISSUE TITLE
[QUERY] Add data disk with write accelerator enabled

### ISSUE THREAD

**Issue by MSSedusch** [Azure/azure-sdk-for-java#42242]
**Query/Question**
Is it possible to attach an existing managed disk to a virtual machine with write accelerator enabled? I had a look at the source code but I could not find a way other than switching to the service clients.

***Why is this not a Bug or a feature Request?***
I might have missed something - so wanted to ask first if this is really not possible today.

**Setup (please complete the following information if applicable):**
 - OS: Windows
 - IDE: Eclipse
 - Library/Libraries: azure-resourcemanager-compute 2.40.0
 
 **Information Checklist**
 Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [x] Query Added
- [x] Setup information Added

**Comment by weidongxu-microsoft** [comment]
@MSSedusch 

We haven't put this property in convenience API. That might be the reason you didn't find it in SDK.

Are you configuring this for Virtual Machine, or Virtual Machine Scale Set?

**Comment by MSSedusch** [comment]
@weidongxu-microsoft I would like to use it for Virtual Machines. I switched back to using the service clients but it would be great to be able to use it in the convenience API. Thanks

**Comment by weidongxu-microsoft** [comment]
Got it.

Status update: I've made a PR https://github.com/Azure/azure-sdk-for-java/pull/42796, but I am not able to test it -- our test subscription does not have any MS familiar vCore. I will need to find a way to get some quota.

**Comment by MSSedusch** [comment]
No, only M Series supports Write Accelerator. Thanks for the PR!

**Comment by alzimmermsft** [comment]
Closing as fixed by #42796
